environments/production/terraform.yml

```diff
@@ -110,6 +110,17 @@ servers:
     group: "couchdb2_proxy"
     os: ubuntu_pro_bionic
 
+  - server_name: "couch11-production"
+    server_instance_type: c5a.24xlarge
+    network_tier: "db-private"
+    az: "b"
+    volume_size: 60
+    volume_encrypted: no
```

Comment:
should be `volume_encrypted: yes` here too.

Reply:
Danny, I will discuss this with you once.

Reply:
@dannyroberts I'm sorry missed this notification. 
yes, the root volume also has to be encrypted but I didn't notice the volume_encrypted parameter is in two places.
Yesterday, I realized it and looked into ways to encrypt the root volume. Seems like the best possible solution is to - take a snapshot 
- create a new volume from the snapshot 
- stop the instance and attach the new volume as the root volume to the instance.
- Finally, import the changes to the state file.

Reply:
OK it's not urgent, so why don't you just make a ticket for this and we can do it after all other couch work is done

Reply:
Sure, Danny. I will create a ticket for this task.